projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -624,9 +769,23 @@ public function send_wpcom_mail_user_invite( $errors, $update, $user ) {
 					)
 				);
 
-				$status_code = wp_remote_retrieve_response_code( $response );
-				if ( is_wp_error( $response ) || 200 !== $status_code ) {
+				if ( 200 !== wp_remote_retrieve_response_code( $response ) ) {
 					$errors->add( 'invitation_not_sent', __( '<strong>Error</strong>: The user invitation email could not be sent, the user account was not created.', 'jetpack' ) );
+					self::$tracking->record_user_event(
+						'sso_user_new_user_invite_send',
+						array(
+							'success' => 'false',
+						)
+					);
+				} else {
+					$custom_message_sent = isset( $new_user_request['message'] ) ? 'true' : 'false';
+					self::$tracking->record_user_event(
+						'sso_user_new_user_invite_send',
```

Comment:
Could we use a different name for this event or this one is ok? 
Its a bit confusing when you read it